libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -74,6 +79,10 @@ class AP_Scheduler
         uint16_t max_time_micros;
     };
 
+    enum class Options : uint8_t {
+        RecordTaskInfo = 1 << 0
```

Comment:
Traditionally the enum entry names have been all upper-case.

Reply:
Fixed